apps/web/modules/survey/lib/survey.ts

```diff
@@ -0,0 +1,143 @@
+import { Organization, Prisma } from "@prisma/client";
+import { cache as reactCache } from "react";
+import { prisma } from "@formbricks/database";
+import { cache } from "@formbricks/lib/cache";
+import { organizationCache } from "@formbricks/lib/organization/cache";
+import { surveyCache } from "@formbricks/lib/survey/cache";
+import { transformPrismaSurvey } from "@formbricks/lib/survey/utils";
```

Comment:
this util function is still getting imported and used here, pls remove this in favor of doing everything in the module itself 🙏